examples/js/loaders/OBJLoader.js

```diff
@@ -8,6 +8,31 @@ THREE.OBJLoader = function ( manager ) {
 
 	this.materials = null;
 
+	this.regexp = {
```

Comment:
Yeah, I made them public also so that people can override them if they want to hack around certain badly authored models :) Probably not a wide use case but something that might be useful to expose. You'd have to still keep the result capture groups the same in order for the code to keep working.